ArkhamOverlay/Utils/IHasImageButton.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Threading;
+
+namespace ArkhamOverlay.Utils {
+    public interface IHasImageButton {
+        string Name { get; }
+        CardType ImageCardType { get; }
+        ImageSource Image { get; set; }
+        ImageSource ButtonImage { get; set; }
+        byte[] ButtonImageAsBytes { get; set; }
+    }
+
+    public static class HasImageButtonExtensions {
+        private static readonly Dictionary<string, BitmapImage> CardImageCache = new Dictionary<string, BitmapImage>();
```

Comment:
Minor: This isn't great IMO. If we're caching things, (especially big stuff like images) I'd rather have a dedicated class or server. This is probably fine for now, but would be good to make a note of it.

Reply:
Discussed- we'll consider implementing a better caching strategy as a feature in the future. I added an issue for this.